semanticdb/Makefile

```diff
@@ -0,0 +1,11 @@
+path="input/src/main/scala/"
+files=$(shell find $(path) -type f -name '*.scala')
+
+all: %.tasty
```

Comment:
I'd rather not add another build system to our project. Can this be replaced by one or more sbt tasks ?

Reply:
It should be possible

Reply:
I removed the Makefile. I didn't saw that the `*.tasty` files I needed were already generated by `sbt`. I'm now using them directly. However, to run the tests it is still needed to run `sbt compile` inside `semanticdb/input` as described in #5253.

Reply:
semanticdb/input/build.sbt should be merged into our project/Build.scala to remove the need for this.